bigquery/value_test.go

```diff
@@ -729,20 +744,23 @@ func TestStructSaverErrors(t *testing.T) {
 	}
 }
 
-func TestNumericString(t *testing.T) {
+func TestNumericStrings(t *testing.T) {
 	for _, test := range []struct {
-		in   *big.Rat
-		want string
+		in             *big.Rat
+		wantNumeric    string
+		wantBigNumeric string
 	}{
-		{big.NewRat(2, 3), "0.666666667"}, // round to 9 places
-		{big.NewRat(1, 2), "0.500000000"},
-		{big.NewRat(1, 2*1e8), "0.000000005"},
-		{big.NewRat(5, 1e10), "0.000000001"},   // round up the 5 in the 10th decimal place
-		{big.NewRat(-5, 1e10), "-0.000000001"}, // round half away from zero
+		{big.NewRat(2, 3), "0.666666667", "0.66666666666666666666666666666666666667"}, // rounding last digit for both numeric/bignumeric
```

Comment:
Consider adding a name field to the test cases. Then, we don't need these comments and they make test debugging easier.

Reply:
Used the index rather than labelling them with a description.

Reply:
And now it's description-based